registries/_format/sf-binary.md

```diff
@@ -0,0 +1,37 @@
+---
+owner: mikekistler
+issue: 
+description: structured fields byte sequence as defined by `sf-binary` in [RFC 8941]
```

Comment:
I think I've put too much emphasis on the ABNF here, since RFC 8941 explicitly states
> If there is disagreement between the parsing algorithms and ABNF, the specified algorithms take precedence.
```suggestion
description: structured fields byte sequence as defined in [RFC 8941]
```